src/tt.h

```diff
@@ -63,6 +63,20 @@ struct TTEntry {
     int16_t  eval16;
 };
 
+struct TTData: public TTEntry {
+
+    TTData(TTEntry* tte) :
```

Comment:
I feel it's safer to mark this single-parameter constructor as `explicit`, so that no implicit conversions from `TTEntry *` are possible by accident.

Reply:
That is a good idea. Makes the code more robust against errors.